sdk/server-proxies/src/InprocessServerProxy.ts

```diff
@@ -18,6 +18,7 @@ const logger = createLogger("InprocessServerProxy");
 const apiVersion = "2023-07-01";
 const httpMethodPost = "POST";
 const httpMethodDelete = "DELETE";
+const consistentKeyForSendToAll = "keyForSendToAll";
```

Comment:
`keyForSendToAll` could be a valid group name and leads to conflict.
Group name is required to exclude any whitespace. (See method `IsGroupNameValid` in RT). 
Adding a whitespace into the string could avoid the conflict.

Reply:
`keyForSendToAll` can be a group name doesn't matter. As it's only a partition key to choose connection.